File: codingame/old/onboarding.cs
using System;
class Player{
	static void Main(string[] args){
		for(;;){
			String name="";
			int n=int.Parse(Console.ReadLine());
			double mi=9999999;
			for(;n>0;n--){
				string[] inputs = Console.ReadLine().Split(' ');
				string enemy = inputs[0];
				double dist=double.Parse(inputs[1]);
				if(mi>dist){
					mi=dist;
					name=enemy;
				}
			}
			Console.WriteLine(name);
		}
	}
}

File: codingame/old/onboarding.vb
module Player
	sub Main()
		while True
			dim name as String
			dim n as integer=Console.ReadLine()
			dim mi as double=9999999
			for i as integer=0 to n-1
				dim inputs as String()=Console.ReadLine().Split(" ")
				dim enemy as String=inputs(0)
				dim dist as double=inputs(1)
				if mi>dist
					mi=dist
					name=enemy
				end if
			next
			Console.WriteLine(name)
		end while
	end sub
end module